src/components/Main.vue

```diff
@@ -73,10 +79,15 @@ export default {
             rendered: false,
             sidebarStatus: false,
             widthVal: '100%',
-            floatVal: 'none'
+            floatVal: 'none',
+            highlight: "",
         };
     },
+
     methods: {
+        highlightedIssue(issue){
```

Comment:
There should be a verb in this method name. `setHighlightedIssue` or `highlightIssue`?